pykern/pkinspect.py

```diff
@@ -191,6 +194,23 @@ def module_name_split(obj):
     return n.split('.');
 
 
+def module_functions(func_prefix, module=None):
+    """Get all module level functions with a prefix
```

Comment:
"Get all functions in `module` starting with `func_prefix`"

Reply:
b29cdf7